shub/image/init.py

```diff
@@ -98,23 +107,16 @@ def cli(project, base_image, base_deps, add_deps, requirements):
         'requirements': _format_requirements(project_dir, requirements),
     }
     values = {key: value if value else '' for key, value in values.items()}
-    source = Template(DOCKERFILE_TEMPLATE.strip())
+    source = Template(DOCKERFILE_TEMPLATE)
     results = source.substitute(values)
     results = results.replace('\n\n', '\n')
 
     click.echo("The following Dockerfile will be created:\n{}".format(results))
-    valid = {"yes": True, "y": True, "ye": True,
-             "no": False, "n": False}
-    while True:
-        dockefile_path = os.path.join(project_dir, 'Dockerfile')
-        choice = input("Save to {}: (y/n)".format(dockefile_path)).lower()
-        if choice in valid:
-            if valid[choice]:
-                with open(dockefile_path, 'w') as dockerfile:
-                    dockerfile.write(results)
-                click.echo('Saved.')
-            break
-        click.echo("Please respond with 'yes'('y') or 'no'(n)")
+    dockefile_path = os.path.join(project_dir, 'Dockerfile')
+    if click.confirm("Save to {}?".format(dockefile_path)):
```

Comment:
Good point. Since there's no configuration questions being asked before the Dockerfile is generated there's no use in having a "dry run" functionality. Just fail directly at the beginning when `Dockerfile` already exists, otherwise write it without extra confirmation.